src/js/containers/bulkDownload/accounts/AccountDataContainer.jsx

```diff
@@ -77,6 +77,14 @@ export class AccountDataContainer extends React.Component {
             this.federalAccountListRequest.cancel();
         }
 
+        // This assures that the federalAccounts state will be disabled
```

Comment:
I don't think this is necessary if you make the suggested change in `AgencyFilter.jsx`